diff -r -U 3 src/minecraft/net/minecraft/server/integrated/IntegratedServer.java src 2/minecraft/net/minecraft/server/integrated/IntegratedServer.java
--- src/minecraft/net/minecraft/server/integrated/IntegratedServer.java 2015-02-20 10:21:36.000000000 +0000
+++ src 2/minecraft/net/minecraft/server/integrated/IntegratedServer.java 2015-02-19 21:34:38.000000000 +0000
@@ -120,6 +120,7 @@
                 this.worldServers[var9] = (WorldServer)(new WorldServerMulti(this, var7, var10, this.worldServers[0], this.theProfiler)).init();
             }
 
+            com.irtimaled.bbor.BoundingBoxOutlineReloaded.worldLoaded(this.worldServers[var9]);
             this.worldServers[var9].addWorldAccess(new WorldManager(this, this.worldServers[var9]));
         }
 
