diff -r -u a/net/minecraft/client/Minecraft.java b/net/minecraft/client/Minecraft.java
--- a/net/minecraft/client/Minecraft.java 2017-09-27 12:01:40.000000000 -0700
+++ b/net/minecraft/client/Minecraft.java 2017-09-27 11:30:01.000000000 -0700
@@ -643,6 +643,8 @@
         }
 
         this.renderGlobal.makeEntityOutlineShader();
+
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.init();
     }
 
     /**
@@ -2276,6 +2278,8 @@
             this.gameSettings.smoothCamera = !this.gameSettings.smoothCamera;
         }
 
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.keyPressed();
+
         for (int i = 0; i < 9; ++i)
         {
             boolean flag = this.gameSettings.keyBindSaveToolbar.isKeyDown();
