diff -r -u a/net/minecraft/client/settings/KeyBinding.java b/net/minecraft/client/settings/KeyBinding.java
--- a/net/minecraft/client/settings/KeyBinding.java 2017-09-27 12:01:40.000000000 -0700
+++ b/net/minecraft/client/settings/KeyBinding.java 2017-09-27 11:52:47.000000000 -0700
@@ -184,5 +184,6 @@
         CATEGORY_ORDER.put("key.categories.multiplayer", Integer.valueOf(5));
         CATEGORY_ORDER.put("key.categories.ui", Integer.valueOf(6));
         CATEGORY_ORDER.put("key.categories.misc", Integer.valueOf(7));
+        CATEGORY_ORDER.put("Bounding Box Outline Reloaded", Integer.valueOf(0));
     }
 }
