diff -r -U 3 mcp940-pre1/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java mcp940-pre1_1.12/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java
--- mcp940-pre1/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java 2017-07-04 17:36:38.000000000 -0700
+++ mcp940-pre1_1.12/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java  2017-07-04 16:04:51.000000000 -0700
@@ -320,6 +320,7 @@
     public void sendQuittingDisconnectingPacket()
     {
         this.connection.getNetworkManager().closeChannel(new TextComponentString("Quitting"));
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.playerDisconnectedFromServer();
     }
 
     /**
