diff -r -u a/net/minecraft/world/chunk/Chunk.java b/net/minecraft/world/chunk/Chunk.java
--- a/net/minecraft/world/chunk/Chunk.java  2017-09-27 12:01:41.000000000 -0700
+++ b/net/minecraft/world/chunk/Chunk.java  2017-09-27 11:34:42.000000000 -0700
@@ -935,6 +935,7 @@
         {
             this.world.loadEntities(classinheritancemultimap);
         }
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.chunkLoaded(this);
     }
 
     /**
