diff -r -u a/net/minecraft/client/renderer/EntityRenderer.java b/net/minecraft/client/renderer/EntityRenderer.java
--- a/net/minecraft/client/renderer/EntityRenderer.java 2017-09-27 12:01:40.000000000 -0700
+++ b/net/minecraft/client/renderer/EntityRenderer.java 2017-09-27 11:53:37.000000000 -0700
@@ -1483,6 +1483,8 @@
             this.renderCloudsCheck(renderglobal, partialTicks, pass, d0, d1, d2);
         }
 
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.render(partialTicks);
+
         this.mc.mcProfiler.endStartSection("hand");
 
         if (this.renderHand)
