diff -r -u a/net/minecraft/server/integrated/IntegratedServer.java b/net/minecraft/server/integrated/IntegratedServer.java
--- a/net/minecraft/server/integrated/IntegratedServer.java 2017-09-27 12:01:41.000000000 -0700
+++ b/net/minecraft/server/integrated/IntegratedServer.java 2017-09-27 11:37:50.000000000 -0700
@@ -114,6 +114,8 @@
                 this.worlds[i] = (WorldServer)(new WorldServerMulti(this, isavehandler, j, this.worlds[0], this.profiler)).init();
             }
 
+            com.irtimaled.bbor.BoundingBoxOutlineReloaded.worldLoaded(this.worlds[i]);
+
             this.worlds[i].addEventListener(new ServerWorldEventHandler(this, this.worlds[i]));
         }
 
