diff -r -U 3 mcp931/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java mcp931 2/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java
--- mcp931/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java  2016-08-27 18:58:30.000000000 +0100
+++ mcp931 2/src/minecraft/net/minecraft/client/multiplayer/WorldClient.java  2016-08-27 18:56:55.000000000 +0100
@@ -320,7 +320,6 @@
     public void sendQuittingDisconnectingPacket()
     {
         this.connection.getNetworkManager().closeChannel(new TextComponentString("Quitting"));
-        com.irtimaled.bbor.BoundingBoxOutlineReloaded.playerDisconnectedFromServer();
     }
 
     /**
