diff -r -U 3 mcp940-pre1/src/minecraft/net/minecraft/network/NetworkManager.java mcp940-pre1_1.12/src/minecraft/net/minecraft/network/NetworkManager.java
--- mcp940-pre1/src/minecraft/net/minecraft/network/NetworkManager.java 2017-07-04 17:36:38.000000000 -0700
+++ mcp940-pre1_1.12/src/minecraft/net/minecraft/network/NetworkManager.java  2017-07-04 17:54:02.000000000 -0700
@@ -32,6 +32,7 @@
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import javax.annotation.Nullable;
 import javax.crypto.SecretKey;
+import net.minecraft.network.login.client.CPacketLoginStart;
 import net.minecraft.util.CryptManager;
 import net.minecraft.util.ITickable;
 import net.minecraft.util.LazyLoadBase;
@@ -188,6 +189,10 @@
                 this.readWriteLock.writeLock().unlock();
             }
         }
+
+        if (packetIn instanceof CPacketLoginStart) {
+            com.irtimaled.bbor.BoundingBoxOutlineReloaded.playerConnectedToServer(this);
+        }
     }
 
     public void sendPacket(Packet<?> packetIn, GenericFutureListener <? extends Future <? super Void >> listener, GenericFutureListener <? extends Future <? super Void >> ... listeners)
