diff -r -u a/net/minecraft/client/multiplayer/WorldClient.java b/net/minecraft/client/multiplayer/WorldClient.java
--- a/net/minecraft/client/multiplayer/WorldClient.java 2017-09-27 12:01:40.000000000 -0700
+++ b/net/minecraft/client/multiplayer/WorldClient.java 2017-09-27 11:53:37.000000000 -0700
@@ -320,6 +320,7 @@
     public void sendQuittingDisconnectingPacket()
     {
         this.connection.getNetworkManager().closeChannel(new TextComponentString("Quitting"));
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.playerDisconnectedFromServer();
     }
 
     /**
