diff -r -U 3 mcp940-pre1/src/minecraft/net/minecraft/client/Minecraft.java mcp940-pre1_1.12/src/minecraft/net/minecraft/client/Minecraft.java
--- mcp940-pre1/src/minecraft/net/minecraft/client/Minecraft.java 2017-07-04 17:36:37.000000000 -0700
+++ mcp940-pre1_1.12/src/minecraft/net/minecraft/client/Minecraft.java  2017-07-04 16:03:22.000000000 -0700
@@ -633,6 +633,8 @@
         }
 
         this.renderGlobal.makeEntityOutlineShader();
+
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.init();
     }
 
     private void func_193986_ar()
@@ -2242,6 +2244,8 @@
             this.gameSettings.smoothCamera = !this.gameSettings.smoothCamera;
         }
 
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.keyPressed();
+
         for (int i = 0; i < 9; ++i)
         {
             boolean flag = this.gameSettings.field_193629_ap.isKeyDown();
