diff -r -U 3 src/minecraft/com/irtimaled/bbor/CommonProxy.java src copy/minecraft/com/irtimaled/bbor/CommonProxy.java
--- src/minecraft/com/irtimaled/bbor/CommonProxy.java 2015-02-23 14:01:07.000000000 +0000
+++ src copy/minecraft/com/irtimaled/bbor/CommonProxy.java  2015-02-23 16:05:40.000000000 +0000
@@ -25,14 +25,14 @@
         if (chunkProvider instanceof ChunkProviderServer) {
             chunkProvider = ReflectionHelper.getPrivateValue(ChunkProviderServer.class, (ChunkProviderServer) chunkProvider, IChunkProvider.class);
             setWorldData(new WorldData(world.getSeed(), world.getWorldInfo().getSpawnX(), world.getWorldInfo().getSpawnZ()));
-            int dimensionId = world.provider.getDimensionId();
+            int dimensionId = world.provider.dimensionId;
             Logger.info("create world dimension: %d, %s (chunkprovider: %s) (seed: %d)", dimensionId, world.getClass().toString(), chunkProvider.getClass().toString(), worldData.getSeed());
             boundingBoxCacheMap.put(dimensionId, new DimensionProcessor(eventHandler, configManager, world, dimensionId, chunkProvider));
         }
     }
 
     public void chunkLoaded(Chunk chunk) {
-        int dimensionId = chunk.getWorld().provider.getDimensionId();
+        int dimensionId = chunk.worldObj.provider.dimensionId;
         if (boundingBoxCacheMap.containsKey(dimensionId)) {
             boundingBoxCacheMap.get(dimensionId).refresh();
         }
