diff -r -U 3 src/minecraft/com/irtimaled/bbor/DimensionProcessor.java src copy/minecraft/com/irtimaled/bbor/DimensionProcessor.java
--- src/minecraft/com/irtimaled/bbor/DimensionProcessor.java  2015-02-23 15:07:53.000000000 +0000
+++ src copy/minecraft/com/irtimaled/bbor/DimensionProcessor.java 2015-02-23 16:05:40.000000000 +0000
@@ -75,10 +75,6 @@
                 structureMap.put(StructureType.WitchHut, getStructuresWithComponent(getStructures(chunkProvider, MapGenScatteredFeature.class), ComponentScatteredFeaturePieces.SwampHut.class));
             }
 
-            if (configManager.drawOceanMonuments.getBoolean()) {
-                structureMap.put(StructureType.OceanMonument, getStructures(chunkProvider, StructureOceanMonument.class));
-            }
-
             if (configManager.drawStrongholds.getBoolean()) {
                 structureMap.put(StructureType.Stronghold, getStructures(chunkProvider, MapGenStronghold.class));
             }
@@ -129,10 +125,10 @@
         }
 
         if (configManager.drawVillages.getBoolean() &&
-                (world.getVillageCollection() != null)) {
+                (world.villageCollectionObj != null)) {
 
             Map<Integer, BoundingBoxVillage> villageBoundingBoxes = new HashMap<Integer, BoundingBoxVillage>();
-            List<Village> villages = world.getVillageCollection().getVillageList();
+            List<Village> villages = world.villageCollectionObj.getVillageList();
             for (Village village : villages) {
                 int villageId = village.hashCode();
                 BlockPos center = village.getCenter();
@@ -160,7 +156,8 @@
     private Set<BlockPos> getDoorsFromVillage(Village village) {
         Set<BlockPos> doors = new HashSet<BlockPos>();
         for (Object doorInfo : village.getVillageDoorInfoList()) {
             VillageDoorInfo villageDoorInfo = (VillageDoorInfo) doorInfo;
-            doors.add(villageDoorInfo.getDoorBlockPos());
+            doors.add(new BlockPos(villageDoorInfo.posX, villageDoorInfo.posY, villageDoorInfo.posZ));
         }
         return doors;
     }
