diff -r -U 3 src/minecraft/net/minecraft/client/Minecraft.java src copy/minecraft/net/minecraft/client/Minecraft.java
--- src/minecraft/net/minecraft/client/Minecraft.java 2015-02-23 16:20:20.000000000 +0000
+++ src copy/minecraft/net/minecraft/client/Minecraft.java  2015-02-22 21:24:58.000000000 +0000
@@ -639,6 +639,7 @@
             this.gameSettings.enableVsync = false;
             this.gameSettings.saveOptions();
         }
+        com.irtimaled.bbor.BoundingBoxOutlineReloaded.init();
     }
 
     public boolean func_152349_b()
@@ -1963,6 +1964,7 @@
                 }
             }
 
+            com.irtimaled.bbor.BoundingBoxOutlineReloaded.keyPressed();
             for (var9 = 0; var9 < 9; ++var9)
             {
                 if (this.gameSettings.keyBindsHotbar[var9].isPressed())
