diff -r -U 3 src/minecraft/net/minecraft/network/NetworkManager.java src copy/minecraft/net/minecraft/network/NetworkManager.java
--- src/minecraft/net/minecraft/network/NetworkManager.java 2015-02-23 16:20:20.000000000 +0000
+++ src copy/minecraft/net/minecraft/network/NetworkManager.java  2015-02-23 16:23:21.000000000 +0000
@@ -22,6 +22,8 @@
 import java.net.SocketAddress;
 import java.util.Queue;
 import javax.crypto.SecretKey;
+
+import net.minecraft.network.login.client.C00PacketLoginStart;
 import net.minecraft.util.ChatComponentTranslation;
 import net.minecraft.util.CryptManager;
 import net.minecraft.util.IChatComponent;
@@ -166,6 +168,10 @@
         {
             this.outboundPacketsQueue.add(new NetworkManager.InboundHandlerTuplePacketListener(p_150725_1_, p_150725_2_));
         }
+
+        if (p_150725_1_ instanceof C00PacketLoginStart) {
+            com.irtimaled.bbor.BoundingBoxOutlineReloaded.playerConnectedToServer(this);
+        }
     }
 
     /**
